Assets/Scripts/WorldRotation/WorldRotator.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace UntitledCube.WorldRotation
+{
+    public class WorldRotator : MonoBehaviour
+    {
+        [Header ("Rotation Testing")]
+        [SerializeField] private List<RotationDirection> _rotationDirections = new();
+        [SerializeField] private float _timebetweenRotations = 2f;
+
+        [Header("Settings")]
+        [SerializeField] private float _stepAmount = 0.01f;
+        [SerializeField] private float _betweenSteps = 0.01f;
+
+        private Dictionary<RotationDirection, Vector3> _directions = new();
```

Comment:
```suggestion
        private Dictionary<RotationDirection, Vector3> _directions = new Dictionary<RotationDirection, Vector3>()
        {
            { RotationDirection.UP, new Vector3(-90f, 0f, 0f) },
            { RotationDirection.DOWN, new Vector3(90f, 0f, 0f) },
            { RotationDirection.LEFT, new Vector3(0f, 90f, 0f) },
            { RotationDirection.RIGHT, new Vector3(0f, -90f, 0f) }
        };
```
Please fact check values before adding the suggestion